### PR TITLE
add cron_log parser

### PR DESCRIPTION
**Pull request by reidliu41**
Signed-off-by: reid_liu <guliu@redhat.com>

### CODE REVIEW

File: insights/parsers/cron_log.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""
```
Comment (bfahr) [inline insights/parsers/cron_log.py]: @reidliu41 please add the file `docs/shared_parsers_catalog/cron_log.rst` as documented [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring) to include your parser in the documentation build.

File: insights/parsers/cron_log.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    Examples:
+        >>> cronlog = shared[CronLog]
+        >>> cronlog.get('Normal exit')
+        [{'timestamp':'Feb  9 03:19:03',
+          'hostname':'dev7u7',
+          'procname': 'anacron[30728]',
+          'message': 'Normal exit (1 job run)',
+          'raw_message': 'Feb  9 03:19:03 dev7u7 anacron[30728]: Normal exit (1 job run)'
+        }]
```
Comment (bfahr) [inline insights/parsers/cron_log.py]: This is the deprecated style of example (using `shared`), please follow the documentation guidelines [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#examples) to document and test your examples.

**Comment by reidliu41** [comment]
@bfahr updated

**Comment by bfahr** [comment]
@reidliu41 you need to signoff `-s` on each commit to pass the DCO test.

File: insights/parsers/tests/test_cron_log.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    print(crond_list[0].get('timestamp'))
+    assert crond_list[0].get('timestamp') == "Feb  9 03:20:01"
+    assert crond_list[4].get('procname') == "CROND[5122]"
+    anacron_list = msg_info.get('anacron[30728]')
+    assert 2 == len(anacron_list)
+    assert anacron_list[0].get('procname') == "anacron[30728]"
+    assert anacron_list[1].get(
+        'raw_message') == "Feb  9 03:19:03 dev7u7 anacron[30728]: Normal exit (1 job run)"
+    assert anacron_list[1].get('message') == "Normal exit (1 job run)"
+    assert anacron_list[1].get('hostname') == "dev7u7"
```
Comment (bfahr) [inline insights/parsers/tests/test_cron_log.py]: @reidliu41 please include testing for your examples as documented [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-examples).

**Comment by bfahr** [comment]
@reidliu41 this data will not be collected by the Insights client, if there is not another reason to add this parser then I would like to close it.

**Comment by reidliu41** [comment]
@bfahr got it, please close it

**Comment by xiangce** [comment]
Close it per the discussion.

**Pull request closed without merging**